Documentation/Signoffs/Power System.md

```diff
@@ -0,0 +1,33 @@
+## Required Format and Sections of the signoff request
+
+This signoff is for the power system of a robot. It must take in the voltage and current supplied by the battery and transform it to the desired levels of each of the subsystems connected. Further, it must appropriately filter the power such that the ripple does not cause the voltage to go out spec for any of the components.
+
+
+## Constraints
+
+Voltage 
+
+
+
+<p align="center">
+<img src="https://i.stack.imgur.com/WK2zg.png">
+</p>
+
+
+## Buildable pdf schematic 
+
+The third section should show the buildable schematic directly embedded in the markdown file as an image (pdf files can't be embedded, but they can be converted with high resolution to jpg). If the schematic is not clearly readable and appropriately sized, the supervisor will reject the signoff. 
+
+The schematic must be appropriate to the design. ie. 3d model for a physical system or wiring schematic for a circuit. Further, the schematic(s) must contain every detail necessary for the design to be built by someone who has no knowledge of the design. Every relevant component value and measurement must be given.
+
+## Analysis
```

Comment:
Needs more analytical verification of the voltage and current output and the ripple in each subsystem